content/blog/2024-06-27-using-bitnami-valkey-chart/index.md

```diff
@@ -0,0 +1,187 @@
+
+First, it is needed to create the configuration for the WordPress deployment, in this case, Valkey’s parameters and the user credential for the site administrator are set.
+
+```bash
+$ cat > settings.yaml <<EOF
+wordpressPassword: "wp_pwd"
+wordpressExtraConfigContent: |
+  define( 'WP_REDIS_HOST', 'myvalkey-master.valkey' );
+  define( 'WP_REDIS_PORT', 6379 );
+  define( 'WP_REDIS_PASSWORD', 'test_pwd' );
+  define( 'WP_REDIS_PREFIX', 'wp_redis' );
+EOF
+```
+
+Once you have the settings parameters, it is time to deploy WordPress in its namespace
+
+```bash
+$ kubectl create namespace wordpress
+$ helm install mywp bitnami/wordpress -f settings.yaml --namespace wordpress
+```
+
+Wait for the deployment to complete and check that the service has been assigned an external IP
+
+```bash
+$ kubectl get svc --namespace wordpress
+NAME                   TYPE         CLUSTER-IP     EXTERNAL-IP  PORT(S)         AGE
+service/mywp-mariadb   ClusterIP    10.106.11.191  <none>       3306/TCP                    74s
+service/mywp-wordpress LoadBalancer 10.100.136.181 192.168.49.50   80:31116/TCP,443:31119/TCP   74s
+```
+
+Using the external IP, browse to the admin page and log in using the previously set credentials.
+
+You will be able to see the WordPress administrator UI where you can search for `Redis Object Cache` plugin, and then install and activate it. After enabling it, wait for some metrics to appear:
+
+![Redis Object Cache Metrics](images/using-bitnami-valkey-chart_4.png)
+
+This is only a drop in the bucket of what can be done with Valkey. Thereafter you could easily integrate Valkey into your projects.
+
+# Support and resources
+
+The Bitnami package for Valkey is available in both the community version, through the [Bitnami GitHub repository](https://github.com/bitnami/charts/tree/main/bitnami/valkey/#installing-the-chart), as well as the enterprise version, [Tanzu Application Catalog](https://app-catalog.vmware.com/catalog/f1242d16-218e-4773-8856-adcb2b2006e9/branch/1542f88a-71c4-42ed-b79c-89bd2063ac9a). Learn more about the differences between these two catalogs in this [blog post](https://tanzu.vmware.com/content/blog/open-source-vs-enterprise-edition-of-vmware-bitnami-application-catalog).
+
+To solve the problems you may have with the Bitnami community packages—including deployment support, operational support, and bug fixes—please open an issue in the Bitnami [Helm charts](https://github.com/bitnami/charts) or [containers](https://github.com/bitnami/containers) GitHub repositories. Also, if you want to contribute to the catalog, feel free to send us a [pull request](https://github.com/bitnami/containers/pulls), and the team will check it and guide you in the process for a successful merge.
+
+If you are interested in learning more about the Tanzu Application Catalog in general, check out the [product webpage](https://tanzu.vmware.com/application-catalog) and [contact us](https://tanzu.vmware.com/application-catalog#get-started).
```

Comment:
When I click on "contact us" I get the same page on the same location as the product webpage. Is that right link?

Also, might suggest using something that 'contact us' - could be a little confusing if 'us' means VMware/Broadcom or Valkey.

Reply:
I am removing that link. 
I think it would be better just to point who is interested to the Tanzu site, so there they could find the `Contact us` and other sections. Just in case that site got updated and the links break.